<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoControlsForm</class>
 <widget class="QWidget" name="VideoControlsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="infoFrame" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="infoLabels">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="frameLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>- / -</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>00.00.00 / 00.00.00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameNavigation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="changeFrameLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Переход к кадру</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="changeFrameSpinBox">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="controlButtonWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5"/>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionPlayPause">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/mediaIcons/play.png</normaloff>:/mediaIcons/play.png</iconset>
   </property>
   <property name="text">
    <string>PlayPause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/mediaIcons/stop.png</normaloff>:/mediaIcons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionNextFrame">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/mediaIcons/right1.png</normaloff>:/mediaIcons/right1.png</iconset>
   </property>
   <property name="text">
    <string>Next frame</string>
   </property>
  </action>
  <action name="actionPrevFrame">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/mediaIcons/left1.png</normaloff>:/mediaIcons/left1.png</iconset>
   </property>
   <property name="text">
    <string>Prev Frame</string>
   </property>
  </action>
  <action name="actionEndVideo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/mediaIcons/right.png</normaloff>:/mediaIcons/right.png</iconset>
   </property>
   <property name="text">
    <string>End Video</string>
   </property>
  </action>
  <action name="actionBeginVideo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/mediaIcons/left.png</normaloff>:/mediaIcons/left.png</iconset>
   </property>
   <property name="text">
    <string>beginVideo</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/mediaIcons/return.png</normaloff>:/mediaIcons/return.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionRepeatFragment">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/mediaIcons/repeat.png</normaloff>:/mediaIcons/repeat.png</iconset>
   </property>
   <property name="text">
    <string>RepeatFragment</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
